docs/release-notes/release-notes-0.14.0.md

```diff
@@ -479,6 +480,8 @@ messages directly. There is no routing/path finding involved.
 * [Fixes two issues around configuration parsing and error
   logging](https://github.com/lightningnetwork/lnd/pull/5948).
 
+* [Refactored itest to better manage contexts inside integration tests](https://github.com/lightningnetwork/lnd/pull/5756).
```

Comment:
nit: update this to `release-notes-0.15.0.md`?

Reply:
Done!